doc/client.rst

```diff
@@ -587,6 +587,38 @@ Single-Record Transactions
     .. index::
         single: User Defined Functions
 
+Multi-Record Transactions
+--------------------------
+
+.. class:: Client
+    :noindex:
+
+    .. method:: commit(transaction: aerospike.Transaction, get_commit_status: bool = False) -> Optional[int]:
+
+        Attempt to commit the given multi-record transaction. First, the expected record versions are
+        sent to the server nodes for verification. If all nodes return success, the transaction is
+        committed. Otherwise, the transaction is aborted.
+
+        Requires server version 8.0+
+
+        :param transaction: Multi-record transaction.
+        :type transaction: :py:class:`aerospike.Transaction`
+        :param bool get_commit_status:
+            If :py:obj:`True`, return one of :ref:`mrt_commit_status_constants`.
+            Otherwise, return :py:obj:`None`.
+
+    .. method:: abort(transaction_id: aerospike.Transaction) -> Optional[int]:
```

Comment:
Doesn't this also have a `get_abort_status` arg?

Reply:
Fixed this